SimPEG/Mesh/DiffOperators.py

```diff
@@ -401,14 +415,19 @@ def fget(self):
         return locals()
     cellGrady = property(**cellGrady())
 
+    def _cellGradzStencil(self):
+        if self.dim < 3: return None
+        BC = ['neumann', 'neumann']
```

Comment:
just a note. boundary conditions are different here vs in the main cellGrad call?